src/difference.js

```diff
@@ -4,7 +4,7 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Finds the set (i.e. no duplicates) of all elements in the first list not
- * contained in the second list.
+ * contained in the second list (in terms of R.contains).
```

Comment:
> is there any circumstance you can imagine where the equality checking is going to be different from that of R.equals?

No. And we would certainly want to know if that was the case, so that we could correct it.